l10n_es_aeat_vat_special_prorrate/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright NuoBiT Solutions, S.L. (<https://www.nuobit.com>)
+# Eric Antones <eantones@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    "name": "AEAT - Prorrata especial de IVA",
+    "summary": "Módulo para gestionar la prorrata especial del IVA "
+    "en las facturas de la AEAT",
+    "version": "14.0.1.0.0",
+    "category": "Accounting",
+    "author": "NuoBiT Solutions, S.L., Eric Antones",
+    "website": "https://github.com/nuobit/odoo-addons",
+    "license": "AGPL-3",
+    "depends": [
+        "l10n_es_aeat_mod303",
+        "l10n_es_special_prorate",
```

Comment:
```suggestion
        "l10n_es_special_prorate",
```
🤔

Reply:
> thinking

l10n_es_aeat_mod303 is needed to compute the prorate percent

Reply:
Then why not fuse `l10n_es_aeat_mod303_special_prorate` into `l10n_es_aeat_vat_special_prorrate`? Both have same dependencies.